csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -432,45 +459,54 @@ public ObjectPrx addFacet(Object servant, Identity identity, string facet)
-    /// addFacetWithUUID with an empty facet.
+    /// Adds a servant to this object adapter's Active Servant Map (ASM), using an automatically generated UUID as
+    /// its identity. Also specifies a facet.
     /// </summary>
     /// <param name="servant">The servant to add.</param>
-    /// <param name="facet">The facet. An empty facet means the default facet.</param>
-    /// <returns>A proxy that matches the generated UUID identity, facet, and this object adapter.</returns>
+    /// <param name="facet">The facet of the Ice object that is implemented by the servant. An empty facet means the
+    /// default facet.</param>
+    /// <returns>A proxy with the generated UUID identity and the specified facet.</returns>
     public ObjectPrx addFacetWithUUID(Object servant, string facet)
     {
         var ident = new Identity(Guid.NewGuid().ToString(), "");
         return addFacet(servant, ident, facet);
     }
 
     /// <summary>
-    /// Add a default servant to handle requests for a specific category.
-    /// Adding a default servant for a category for
-    /// which a default servant is already registered throws AlreadyRegisteredException. To dispatch operation
-    /// calls on servants, the object adapter tries to find a servant for a given Ice object identity and facet in the
+    /// Adds a default servant to handle requests for a specific category. When an object adapter dispatches an
+    /// incoming request, it tries to find a servant for the identity and facet carried by the request in the
     /// following order:
-    ///
-    /// The object adapter tries to find a servant for the identity and facet in the Active Servant Map.
-    /// If no servant has been found in the Active Servant Map, the object adapter tries to find a default servant
-    /// for the category component of the identity.
-    /// If no servant has been found by any of the preceding steps, the object adapter tries to find a default
-    /// servant for an empty category, regardless of the category contained in the identity.
-    /// If no servant has been found by any of the preceding steps, the object adapter gives up and the caller
-    /// receives ObjectNotExistException or FacetNotExistException.
+    /// <list type="bullet">
+    /// <item>The object adapter tries to find a servant for the identity and facet in the Active Servant Map.</item>
+    /// <item>If this fails, the object adapter tries to find a default servant for the category component of the
+    /// identity.</item>
+    /// <item>If this fails, the object adapter tries to find a default servant for the empty category, regardless of
+    /// the category contained in the identity.</item>
+    /// <item>If this fails, the object adapter tries to find a servant locator for the category component of the
+    /// identity. If there is no such servant locator, the object adapter tries to find a servant locator for the
+    /// empty category.
+    /// <list type="bullet">
```

Comment:
Is this nested list intentional?

Reply:
Ah, I was looking at the list above this one.
This is intentional, to match the list we have in C++:
```
...
        ///  - If this fails, the object adapter tries to find a servant locator for the category component of the
        ///    identity. If there is no such servant locator, the object adapter tries to find a servant locator for the
        ///    empty category.
        ///    - If a servant locator is found, the object adapter tries to find a servant using this servant locator.
        ///  - If all the previous steps fail, the object adapter gives up and the caller receives an
        ///    ObjectNotExistException or a FacetNotExistException.
...
```
Could probably just dedent the tick-mark and not bother with a nested list though?